### DISCUSSION TITLE
Button sm:square lg:regular

### DISCUSSION THREAD

**Discussion by zerdotre** [livewire/flux#251] [Feature Requests] [answered]
TLDR; I'd like a button to be only square on small screens.

L;
I'm using a button with an icon and some text. I wrapped the text in a span element which is hidden on small screens. So only the icon is displayed, but I'd like the button to also be square on small screens.

**Comment by joshhanley** [comment, marked as answer]
Closing this as there is currently an open issue #824 for this.
